app/views/layouts/admin.html.erb

```diff
@@ -84,6 +84,12 @@
         </div>
       <% end %>
 
+      <div
+        data-controller="error"
+        data-action="error:generate@document->error#generateErrorAlert">
+          <div data-error-target="errorZone"></div>
```

Comment:
```suggestion
          <div data-error-target="errorZone" role="alert"></div>
```
Adding the alert role will make sure the error is immediately announced to screen reader users 🙂